### PR TITLE
fix(sdk): apply selective disclosure to results of get submission requirements

### PR DESCRIPTION
**Pull request by vkubiv**

### CODE REVIEW

File: pkg/credentialquery/credentialquery.go [hunk 1]
```diff
@@ -48,6 +52,14 @@ func WithCredentialReader(credentialReader api.CredentialReader) QueryOpt {
 	}
 }
 
+// WithSelectiveDisclosureApply enables selective disclosure apply.
+func WithSelectiveDisclosureApply(didResolver api.DIDResolver) QueryOpt {
```
[the remark this answers is not in the document]
Comment (vkubiv) [reply pkg/credentialquery/credentialquery.go]: done

**Review by birtony — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-sdk-gomobile/credential/inquirer.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ import (
 // Inquirer implements querying credentials using presentation definition.
 type Inquirer struct {
 	goAPICredentialQuery *credentialquery.Instance
+	goDIDResolver        goapi.DIDResolver
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/credential/inquirer.go:31]:
```suggestion
	goAPIDIDResolver        goapi.DIDResolver
```

Just for consistency with the other field names

File: pkg/credentialquery/credentialquery.go [hunk 3]
```diff
@@ -73,9 +85,22 @@ func (c *Instance) GetSubmissionRequirements(
 		query.InputDescriptors[i].Schema = nil
 	}
 
+	var matchOpts []presexch.MatchRequirementsOpt
+	if qOpts.applySelectiveDisclosure {
+		matchOpts = append(matchOpts,
+			presexch.WithSelectiveDisclosureApply(),
+			presexch.WithSDCredentialOptions(
+				verifiable.WithDisabledProofCheck(),
```
Comment (DRK3) [inline pkg/credentialquery/credentialquery.go:93]: Is it safe to always disable the proof check here?

**Review by DRK3 — APPROVED** [review]
Left a few comments to take a look at, otherwise looks good!

**Comment by DRK3** [comment]
Also, I would suggest shortening the commit title since it overflows into the description

**Pull request merged**